R/util.R

```diff
@@ -107,7 +107,7 @@ file_name_without_shell_chars <- function(file) {
 as_tmpfile <- function(str) {
   if (length(str) > 0) {
     str_tmpfile <- tempfile("rmarkdown-str", fileext = ".html")
-    writeLines(str, str_tmpfile)
+    writeLines(str, str_tmpfile, useBytes =  TRUE)
```

Comment:
The `str` probably needs to be encoded to UTF-8 first if `useBytes = TRUE`, i.e. `enc2utf8(str)`